Added Domain in Excluded Global variable

cs-sujeet-ahirrao opened the pull request:
Added below domains in Excluded List Global Variable

old domain list - "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com"

Updated Domain List: 'google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com,fortinet.com,twitter.com,facebook.com,linkedin.com,instagram.com'

Changes Made:

> Updated -> 11 Demo >> Create Default Global Variables playbook
> Updated -> Global Variables > 'Excludelist_Domains'

cs-sujeet-ahirrao committed:
Added Domain in Excluded globlal variable

Added below domains in Excluded List Global Variable

old domain list - "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com"

Updated Domain List: 'google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com,fortinet.com,twitter.com,facebook.com,linkedin.com,instagram.com'

Changes Made:

> Updated -> 11 Demo >> Create Default Global Variables playbook
> Updated -> Global Variables > 'Excludelist_Domains'

cs-suyog-jinturkar commented:
What all unit testing was carried out to test this change?

cs-rohit-khune commented:
Refer to attached Unit Test
[Mantis ID - 0770430 Unit Test Case Document.docx](https://github.com/fortinet-fortisoar/solution-pack-incident-response/files/7912210/Mantis.ID.-.0770430.Unit.Test.Case.Document.docx)

cs-tushar-kanade commented on playbooks/globalVariables.json:
```diff
@@ -14,8 +14,8 @@
-        "value": "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com",
-        "default_value": "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com"
+        "value": "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com,fortinet.com,twitter.com,facebook.com,linkedin.com,instagram.com",
+        "default_value": "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com,fortinet.com,twitter.com,facebook.com,linkedin.com,instagram.com"
```
add fortiguard.com , forticloud.com  as well

cs-sujeet-ahirrao replied:
@cs-pooja-singh @cs-tushar-kanade Added fortiguard.com , forticloud.com in globalVariable.json and Create Default Global Variable.json

cs-tushar-kanade reviewed (changes requested):
Please do add requested items

cs-sujeet-ahirrao committed:
Added fortiguard.com , forticloud.com in globalVariable.json and Create Default Global Variable.json

Added fortiguard.com , forticloud.com in globalVariable.json and Create Default Global Variable.json

cs-tushar-kanade commented:
@cs-suyog-jinturkar please check and merge
